public/javascripts/application.js

```diff
@@ -362,6 +362,22 @@ jQuery.fn.preventDoubleSubmit = function() {
   });
 };
 
+function scrollToErrorIfFound() {
```

Comment:
Since these two new functions are only used in the LiveValidation JavaScript file, can/should they be moved there?

Reply:
yeah the reason I didn't put them there was because the LiveValidation file is just classes/prototypes so it seemed weird to move them. I can if you think that's better though.

Reply:
Eh, let's move them, then. I think it's better to have as much of the LiveValidation code as possible contained in the LiveValidation file, and it wasn't really our decision to just have classes and prototypes in that file -- that's just the way it came, and we're modifying it anyway.